.github/workflows/publish-release.yml

```
@@ -69,7 +73,7 @@ jobs:
           devCentralToken: ${{ secrets.BALLERINA_DEV_CENTRAL_ACCESS_TOKEN }}
           ballerinaBotWorkflow: $ {{ secrets.BALLERINA_BOT_WORKFLOW }}
         run: |
-          ./gradlew build -Pversion=${VERSION}
+          ./gradlew build -Pversion=${VERSION} -x centralTests -x test
```

```suggestion
          ./gradlew build -Pversion=${VERSION}
```

I hope these were skipped for testing work

---

Yes they were skipped for the purpose of testing I shall include them.